README.md

```diff
@@ -1 +1,37 @@
-# prometheus-edge-hub-operator
\ No newline at end of file
+# prometheus-edge-hub-operator
+
+## Description
+
+Prometheus Edge Hub is a replacement of the Prometheus Pushgateway which allows for the pushing of 
```

Comment:
> Prometheus Edge Hub is a replacement of the Prometheus Pushgateway that provides an endpoint to push metrics for later scraping by Prometheus, rather than having Prometheus scrape the metric sources directly.
>
> Prometheus Edge Hub differs from the Prometheus Pushgateway in several ways, the most important 
of which are:
> 1. Prometheus Edge Hub not overwrite timestamps
> 2. Metrics in Prometheus Edge Hub persist only until scraped

Reply:
done